src/API.php

```diff
@@ -49,6 +49,12 @@ function get_shlinks($args = null) {
 		return $this->request('GET', $endpoint, $args);
 	}
 
+	function get_domains() {
+		$base_url = $this->options->get('base_url');
+		$endpoint = "$base_url/rest/v2/domains";
+		return $this->request('GET', $endpoint);
```

Comment:
This should probably be scoped under wp-shlink to not conflict with other possible APIs. Probably same for the above as well.